components/org.wso2.identity.webhook.wso2.event.handler/src/main/java/org/wso2/identity/webhook/wso2/event/handler/api/builder/WSO2CredentialEventPayloadBuilder.java

```diff
@@ -63,12 +62,15 @@ public EventPayload buildCredentialUpdateEvent(EventData eventData) throws Ident
         Flow flow = IdentityContext.getThreadLocalIdentityContext().getCurrentFlow();
         String action = null;
         String initiatorType = null;
+        String credentialType = Flow.CredentialType.PASSWORD.name();
```

Comment:
Let's avoid initializing with a default value and depend only on the creadential type in the flow.

Reply:
Fixed 0d89e6295d6df89bc51b94611d365b37d549cf21